cypress/support/step-definitions/common-steps.js

```diff
@@ -138,6 +142,14 @@ Given('I open basic Design System {string} component docs page', (component) =>
   visitDocsUrl(component, 'basic', false, 'design-system-');
 });
 
+Given('I open default Design System {string} component docs page', (component) => {
```

Comment:
if we have the same step in line 28, so we probably need to make refactor here :) as I mentioned before

Reply:
fixed